Sources/Random/CryptoRandom.swift

```diff
@@ -6,7 +6,11 @@ public final class CryptoRandom: Random {
 
     public func bytes(_ num: Int) -> Bytes {
         var random = Bytes(repeating: 0, count: num)
-        RAND_bytes(&random, Int32(num))
+		guard RAND_bytes(&random, Int32(num)) == 1 else {
+			// If the requested number of random bytes couldn't be read,
+			// we need to fail fast and hard.
+			abort()
```

Comment:
I think the issue here is that to throw properly, the protocol has to be edited, which would be a breaking change. I think in security situations, these types of things are more acceptable. 

I'm more for figuring out a throw or some other option, possibly trying a few times, then logging. Anything that won't crash is on the table.

@C0deH4cker fill in if I'm missing something.